eodag/utils/__init__.py

```diff
@@ -399,23 +443,11 @@ def __exit__(self, *exc):
         return False
 
 
+@_deprecated(reason="Use ProgressCallback class instead", version="2.2.1")
 class NotebookProgressCallback(ProgressCallback):
     """A custom progress bar to be used inside Jupyter notebooks"""
 
-    def __call__(self, current_size, max_size=None):
-        """Update the progress bar"""
-        if max_size is not None:
-            self.max_size = max_size
-        if self.pb is None:
-            self.pb = tqdm_notebook(
-                total=self.max_size,
-                unit=self.unit,
-                unit_scale=self.unit_scale,
-                desc=self.desc,
-                position=self.position,
-                dynamic_ncols=True,
-            )
-        self.pb.update(current_size)
+    pass
 
 
 def get_progress_callback():
```

Comment:
When used in an Ipython environment, it will still return `NotebookProgressCallback` and emit a deprecated warning.

Reply:
thanks for the feedback @maximlt 
> there's still in the code base this mixed usage of bar = ProgressCalback() and bar = get_progress_callback()

> When used in an Ipython environment, it will still return NotebookProgressCallback and emit a deprecated warning.

- [ ] Deprecate `get_progress_callback` and make it always return `ProgressCalback`

> The docs will need to be updated

- [ ] update the doc and remove custom progress callback usage or make it use `ProgressCalback`

> what's the difference between setup_logging(0) and setup_logging(1)?

Same logging level, with progress-bar enabled when using  `setup_logging(1)`, and disabled when using `setup_logging(0)`